infra/ansible/playbooks/grafana.yaml

```diff
@@ -0,0 +1,143 @@
+- name: Grafana Setup
+  hosts: "{{ host }}"
+
+  tasks:
+  - name: Create keyrings directory for Grafana
+    file:
+      path: /etc/apt/keyrings
+      state: directory
+      mode: '0755'
+
+  - name: Download and install Grafana GPG key
+    shell: 
+      cmd: wget -q -O - https://apt.grafana.com/gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/grafana.gpg
+      creates: /etc/apt/keyrings/grafana.gpg
+    vars: 
+      ansible_ssh_user: "{{ admin_user }}"
+
+  - name: Add Grafana stable repository
+    shell:
+      cmd: echo "deb [signed-by=/etc/apt/keyrings/grafana.gpg] https://apt.grafana.com stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+      creates: /etc/apt/sources.list.d/grafana.list
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+
+  - name: Update apt cache and install Grafana
+    become: true
+    apt:
+      name: grafana
+      state: present
+      update_cache: yes
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+
+  - name: Create /etc/grafana directory
+    become: true
+    file:
+      path: /etc/grafana/
+      state: directory
+      owner: grafana
+      group: grafana
+      mode: '0755'
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+
+  - name: Add grafana config file
+    become: true
+    template:
+      src: grafana/grafana.ini.j2
+      dest: /etc/grafana/grafana.ini
+      owner: grafana
+      group: grafana
+      mode: '0755'
+    vars:
+      ansible_ssh_user: "{{ admin_user }}"
+      grafana_domain: "{{ lookup('ini', 'grafana_domain', file=ini_file) }}"
+      grafana_oath_client_id: "{{ lookup('ini', 'grafana_oath_client_id', file=ini_file) }}"
+      grafana_oath_client_secret: "{{ lookup('ini', 'grafana_oath_client_secret', file=ini_file) }}"
+      grafana_oath_auth_url: "{{ lookup('ini', 'grafana_oath_auth_url', file=ini_file) }}"
+      grafana_oath_token_url: "{{ lookup('ini', 'grafana_oath_token_url', file=ini_file) }}"
+      grafana_oath_api_url: "{{ lookup('ini', 'grafana_oath_api_url', file=ini_file) }}"
+
+  - name: Clone Aligned repository
+    git:
+      repo: https://github.com/yetanotherco/aligned_layer.git
+      dest: /home/{{ ansible_user }}/repos/telemetry/aligned_layer
+      #version: v0.12.0
+      version: staging
```

Comment:
Temporary fix to allow compilation, since `v.012.0` has a missing comma inside a JSON. 

Should be pinned to a future release.